docs/clustering.rst

```diff
@@ -2,7 +2,9 @@
 Clustering
 ==========
 
-One of Fathom's most powerful features is its flexible clustering algorithm, which groups nodes based on their proximity and ancestry. It is documented here as top-level functions but is also available directly within rulesets as :func:`bestCluster`, which has the advantage of letting you direct its results to further rules. Clustering is computationally expensive (at least O(n^2)) but widely applicable.
```

Comment:
This splits off the final sentence ("Clustering is computationally expensive (at least O(n^2)) but widely applicable.") into a separate note, and expands upon it slightly.  Otherwise, the line is untouched.